server/util/courseAvailability.js

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+
+export async function prepareCoursePayload (course) {
+    const { TermCode, SubjectCode, CourseID } = course;
+
+    const courseDataPayload = {
+        TermCode: parseInt(TermCode, 10).toString(),
+        SubjectCode: parseInt(SubjectCode, 10).toString(),
+        CourseID: CourseID,
+    };
+
+    return courseDataPayload;
+};
+
+export async function fetchSingleCourseAvailability (course) {
+    const courseDataPayload = await formatCourseDataPayload(course);
+
+    try {
+        const availabilityResponse = await axios.post('http://localhost:8080', courseDataPayload);
```

Comment:
eventually you will want to move the url into a .env variable instead of hardcoding it to allow for deployment to a hosted service, but for now it's not an issue.  Just something to think about.

Reply:
Sounds good. Will do that soon.